sympy/utilities/lambdify.py

```diff
@@ -350,6 +350,21 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
         When ``True``, ``sympy.simplify.cse`` is used, otherwise (the default)
         the user may pass a function matching the ``cse`` signature.
 
+    docstring_limit : int or None
```

Comment:
I don't know if this needs to be a public option. This is just a convenience anyway, and IMO the only reason this even exists is because str is slow. At best I would just make it a module level variable if someone really wants to override it.